blazor-wasm/WebApplication1.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

At the time being the current version of Blazor supports netstandard2.1
```suggestion
    <TargetFramework>netstandard2.1</TargetFramework>
```